### PR TITLE
i18n(de): translate components/link-buttons

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description

Translation of `components/link-buttons.mdx` into German language.

### CODE REVIEW

**Comment by trueberryless** [comment]
Link fixed in https://github.com/withastro/starlight/pull/2460

File: docs/src/content/docs/de/components/link-buttons.mdx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import { LinkButton } from '@astrojs/starlight/components';
+```
+
+## Verwendung
+
+Verwende die Komponente `<LinkButton>`, um einen visuell eindeutigen Call-to-Action-Link anzuzeigen.
+Ein Link-Button ist nützlich, um Nutzer zu den relevantesten oder handlungsrelevanten Inhalten zu leiten und wird häufig auf Landing Pages verwendet.
+
+Ein `<LinkButton>` benötigt ein [`href`](#href) Attribut.
+Optional kann das Aussehen des Link-Buttons mit dem Attribut [`variant`](#variant) angepasst werden, das auf `primär` (Standard), `sekundär` oder `minimal` gesetzt werden kann.
```
Comment (HiDeoo) [inline docs/src/content/docs/de/components/link-buttons.mdx:32]:
```suggestion
Optional kann das Aussehen des Link-Buttons mit dem Attribut [`variant`](#variant) angepasst werden, das auf `primary` (Standard), `secondary` oder `minimal` gesetzt werden kann.
```

A bit too much translation 😄

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the new translation.

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). Left a suggestion to fix some attribute values but otherwise looks good to me :thumbsup:.

**Review by HiDeoo — APPROVED** [review]
> Thx for the many Hacktoberfest PRs, but I think I have more than 4 by now 😊

Amazing, thanks for participating and choosing the Starlight repo for your contributions! 🎉

**Pull request merged**